Cache/IO.lean

```diff
@@ -87,21 +87,20 @@ def getCurl : IO String := do
 def getLeanTar : IO String := do
   return if (← LEANTARBIN.pathExists) then LEANTARBIN.toString else "leantar"
 
-abbrev PackageDirs := Lean.RBMap String FilePath compare
-
 /--
 `CacheM` stores the following information:
 * the source directory where `Mathlib.lean` lies
-* package directories
+* the Lean search path. This contains
+  paths to the source directory of each imported package. The build files (e.g. `.olean`)
+  of a package should then be in a `.lake`-folder inside this root directory
+  (see `LIBDIR` and `IRDIR`).
```

Comment:
Can you repeat the warning about `lean_lib`s with custom srcPaths here?

Reply:
updated